src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -1,20 +1,10 @@
 package core.basesyntax;
 
 public class FootballMatch {
-
-    /**
-     * <p>С помощью тернарного оператора написать функцию, которая будет считать исход матча.
-     * Метод будет принимать 4 параметра и возвращать целое число.
-     * 4 параметра метода - результат матча и то, какую ставку сделал пользователь.
-     * Пример: 0,2,1,1. Что значит: команды сыграли в счёт 0:2, а пользователь ставил 1:1.
-     * Задача:
-     * если пользователь угадал 100% результат - возвращать 2
-     * если угадал, что выиграла какая-либо команда
-     * (пр. сыграли 1:2, а ставил 0:3 или сыграли 2:2, а ставил 3:3 - то пользователь у выиграше)
-     * то возвращать 1
-     * если не угадал ничего - возвращать 0.</p>
-     */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1 && result2 == bet2) ? 2 :
```

Comment:
please fix checkstyle
run command `mvn clean package` locally

Reply:
Did it few times. Result is the same as yours in video. When I write "mvn clean path" in terminal I get build failure, but no menthions in what line (screen1). And when I click maven -> clean on the right side everything is ok (screen2).
![Screenshot_1](https://user-images.githubusercontent.com/66619756/88538429-07fc7700-d018-11ea-9842-a118be591f52.png)
![Screenshot_2](https://user-images.githubusercontent.com/66619756/88538433-08950d80-d018-11ea-82a5-e661a0e804dd.png)